src/containers/Tables/BidTable/BidTableRow.test.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import { HEADERS } from './BidTable';
+import BidTableRow, { TEST_ID } from './BidTableRow';
+import { ethers } from 'ethers';
+
+const mockDate = new Date(16461998260001);
+
+const mockData = {
+	auctionId: '7124487995',
+	date: mockDate, // 02/03/2022
+	domainId:
+		'0x617b3c878abfceb89eb62b7a24f393569c822946bbc9175c6c65a7d2647c5402',
+	domainMetadataUrl: 'ipfs://QmYKjGMUG3qmx9TZqUSpceHtP2hjomUCiB2fMbJXVBaENd',
+	domainName: 'wilder.cats',
+	highestBid: ethers.utils.parseEther('1000'),
+	yourBid: ethers.utils.parseEther('500'),
+};
+
+const renderComponent = ({ data = mockData } = {}) => {
+	return render(
+		<table>
+			<tbody>
+				<BidTableRow data={data} />
+			</tbody>
+		</table>,
+	);
+};
```

Comment:
I think we can just use `return <BidTableRow data={data} />;` instead of using `<table>, <tbody>` tag since it is the test file of `BidTableRow` component.

Reply:
I tried that, but the test suite wasn't happy rendering a `<tr>` outside of a `<table>` and `<tbody>` - bit annoying